Index: facette/Makefile
===================================================================
--- facette.orig/Makefile	2014-03-12 08:42:49.347425084 +0100
+++ facette/Makefile	2014-03-12 08:43:09.175633651 +0100
@@ -251,9 +251,7 @@
 	done) && $(call mesg_ok) || $(call mesg_fail)
 
 lint-static: $(SCRIPT_OUTPUT)
-	@$(call mesg_start,lint,Checking $(notdir $(SCRIPT_OUTPUT:.js=.src.js)) with JSHint...)
-	-@$(JSHINT) $(JSHINT_ARGS) $(SCRIPT_OUTPUT:.js=.src.js) && \
-		$(call mesg_ok) || $(call mesg_fail)
+	@#PATCH: “Evil” JSHint removal
 
 # Test
 PKG_SRC = $(wildcard pkg/*)
