Index: facette/Makefile
===================================================================
--- facette.orig/Makefile	2014-11-16 12:46:35.548306293 +0100
+++ facette/Makefile	2014-11-16 12:46:35.544306103 +0100
@@ -71,9 +71,7 @@
 	fi
 
 jshint:
-	@if [ -z "$(call path_search,$(JSHINT))" ]; then \
-		$(call npm_install,$(NPM_JSHINT)); \
-	fi
+	@#PATCH: “Evil” JSHint removal
 
 clean: clean-bin clean-doc clean-static clean-test clean-dist
 	@$(call mesg_start,clean,Cleaning source symlink...)
@@ -327,9 +325,7 @@
 	done) && $(call mesg_ok) || $(call mesg_fail)
 
 lint-static: jshint $(SCRIPT_OUTPUT)
-	@$(call mesg_start,lint,Checking $(notdir $(SCRIPT_OUTPUT:.js=.src.js)) with JSHint...)
-	-@$(JSHINT) $(JSHINT_ARGS) $(SCRIPT_OUTPUT:.js=.src.js) && \
-		$(call mesg_ok) || $(call mesg_fail)
+	@#PATCH: “Evil” JSHint removal
 
 # Test
 TEST_DIR = $(BUILD_DIR)/tests
