Index: facette/Makefile
===================================================================
--- facette.orig/Makefile	2014-02-11 00:20:04.037793998 +0100
+++ facette/Makefile	2014-02-11 08:53:17.818392083 +0100
@@ -242,9 +242,7 @@
 		$(call mesg_ok) || $(call mesg_fail)
 
 lint-static: $(SCRIPT_OUTPUT)
-	@$(call mesg_start,lint,Checking $(notdir $(SCRIPT_OUTPUT:.js=.src.js)) with JSHint...)
-	-@$(JSHINT) $(JSHINT_ARGS) $(SCRIPT_OUTPUT:.js=.src.js) && \
-		$(call mesg_ok) || $(call mesg_fail)
+	@#PATCH: “Evil” JSHint removal
 
 # Test
 PKG_SRC = $(wildcard pkg/*)
