Index: facette/Makefile
===================================================================
--- facette.orig/Makefile	2014-02-11 08:53:28.558502554 +0100
+++ facette/Makefile	2014-02-11 08:54:13.554965146 +0100
@@ -25,7 +25,7 @@
 PANDOC_ARGS = --standalone --to man
 
 UGLIFYJS ?= uglifyjs
-UGLIFYSCRIPT_ARGS = --comments --compress --mangle --screw-ie8
+UGLIFYSCRIPT_ARGS =
 
 JSHINT ?= jshint
 JSHINT_ARGS = --show-non-errors
