Index: facette/Makefile
===================================================================
--- facette.orig/Makefile	2014-08-02 10:04:45.317342623 +0200
+++ facette/Makefile	2014-08-02 10:04:45.313342434 +0200
@@ -43,7 +43,7 @@
 PANDOC_ARGS = --standalone --to man
 
 UGLIFYJS ?= uglifyjs
-UGLIFYSCRIPT_ARGS = --comments --compress --mangle --screw-ie8
+UGLIFYSCRIPT_ARGS =
 NPM_UGLIFYJS = uglify-js
 
 JSHINT ?= jshint
