Index: facette/src/style/Makefile
===================================================================
--- facette.orig/src/style/Makefile	2013-12-26 11:30:04.000000000 +0100
+++ facette/src/style/Makefile	2013-12-26 11:33:08.320112465 +0100
@@ -53,7 +53,7 @@
 
 $(CSS_FILE_MIN): $(TEMP_DIR) $(CSS_FILE)
 	@$(call mesg_start,Packing $(notdir $(CSS_FILE)) file...)
-	@$(LESSC) $(LESSC_ARGS) --yui-compress $(CSS_FILE) >$(CSS_FILE_MIN) && $(call mesg_ok) || $(call mesg_fail)
+	@$(LESSC) $(LESSC_ARGS) --compress $(CSS_FILE) >$(CSS_FILE_MIN) && $(call mesg_ok) || $(call mesg_fail)
 
 $(CSS_FILE_PRINT): $(TEMP_DIR) $(SRC_FILES_PRINT)
 	@$(call mesg_start,Merging source files into $(notdir $(CSS_FILE_PRINT))...)
@@ -61,7 +61,7 @@
 
 $(CSS_FILE_PRINT_MIN): $(TEMP_DIR) $(CSS_FILE_PRINT)
 	@$(call mesg_start,Packing $(notdir $(CSS_FILE_PRINT)) file...)
-	@$(LESSC) $(LESSC_ARGS) --yui-compress $(CSS_FILE_PRINT) >$(CSS_FILE_PRINT_MIN) && \
+	@$(LESSC) $(LESSC_ARGS) --compress $(CSS_FILE_PRINT) >$(CSS_FILE_PRINT_MIN) && \
 		$(call mesg_ok) || $(call mesg_fail)
 
 lint:
